sru/modules.py

```diff
@@ -427,7 +438,8 @@ def __init__(self,
                  nn_rnn_compatible_return: bool = False,
                  custom_m: Optional[Union[nn.Module, List[nn.Module]]] = None,
                  proj_input_to_hidden_first: bool = False,
-                 amp_recurrence_fp16: bool = False):
+                 amp_recurrence_fp16: bool = False,
+                 post_ln: bool = False):
```

Comment:
@hpasapp can we change the name from `post_ln` to sth like `post_normalization` or `normalize_after`?
e.g.
https://github.com/pytorch/fairseq/blob/1bc83c703ad70d7f62c1e54b197e29b95d07b1f0/fairseq/modules/transformer_layer.py#L130

Reply:
addressed in b8235f0